### PR TITLE
Update rust-webvr

### PR DESCRIPTION
**Pull request by MortimerGoro**
Required for https://github.com/servo/servo/issues/16556

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [x] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17031)

### CODE REVIEW

**Comment by emilio** [comment]
Dupe of #17023... Hope the queue was faster...

**Comment by emilio** [comment]
Ouch, missed the comment over there

File: components/script/dom/vrdisplay.rs [hunk 1]
```diff
@@ -71,7 +71,9 @@ pub struct VRDisplay {
     frame_data_status: Cell<VRFrameDataStatus>,
     #[ignore_heap_size_of = "channels are hard"]
     frame_data_receiver: DOMRefCell<Option<IpcReceiver<Result<Vec<u8>, ()>>>>,
-    running_display_raf: Cell<bool>
+    running_display_raf: Cell<bool>,
+    paused: Cell<bool>,
```
Comment (emilio) [inline components/script/dom/vrdisplay.rs]: Please add comments about how the state is managed here.

**Review by emilio — COMMENTED** [review]
The rust-webvr bump that added the enums should've probably be a breaking change... But fine. r=me

File: components/script/dom/vrdisplay.rs (hunk 1 above)
Comment (MortimerGoro) [inline components/script/dom/vrdisplay.rs]: Yes, it makes sense. I'll do breaking change for added enums from now on.

**Comment by nox** [comment]
Wait, 0.3.2 needs code change? That should have been 0.4.0 then.

**Comment by MortimerGoro** [comment]
@nox Yes, I didn't though about that because the API change was "just" adding new enum events. I'll do breaking changes for added enums from now on.

**Comment by nox** [comment]
You should yank that bump and publish 0.4 right now.

**Comment by MortimerGoro** [comment]
> You should yank that bump and publish 0.4 right now.

Done

**Comment by nox** [comment]
Perfect, thanks!

**Pull request merged**